src/app/room-creation/room-creation-modal/room-creation-modal.component.ts

```diff
@@ -50,9 +50,15 @@ export class RoomCreationModalComponent implements OnInit {
     this.loadingStatus.isLoading = true;
     this.randomNamesProvider
       .provide(this.formGroup.controls.name.value)
-      .subscribe(randomName => {
-        this.formGroup.controls.name.setValue(randomName);
-        this.loadingStatus.isLoading = false;
+      .subscribe({
+        next: randomName => {
+          this.formGroup.controls.name.setValue(randomName);
+          this.loadingStatus.isLoading = false;
+        },
+        error: _error => {
+          this.formGroup.controls.name.setValue('');
```

Comment:
is that line needed?

Reply:
The error one? That is because the API we use for random names is rate-limted, and the call fails if we request to often :) if that happens, I want the app to work still, just no random name. Is that what you meant?